dashboard/app/controllers/aichat_requests_controller.rb

```diff
@@ -1,14 +1,15 @@
 require 'cdo/throttle'
 
-AICHAT_PREFIX = "aichat/".freeze
-DEFAULT_REQUEST_LIMIT_PER_MIN = 50
-DEFAULT_POLLING_INTERVAL_MS = 1000
-DEFAULT_POLLING_BACKOFF_RATE = 1.2
-
 class AichatRequestsController < ApplicationController
   include AichatSagemakerHelper
   authorize_resource class: false
 
+  AICHAT_REQUEST_COUNT_PREFIX = "aichat/requests/".freeze
```

Comment:
Realizing that constants defined outside of classes in Rails are global, so feels like it might be better to scope them in general. Originally had a shared constant here that I ended up moving so no longer directly relevant, but seems like a good idea anyway. All of these constants are used exclusively in this controller.